### PR TITLE
Create php-nl.html.markdown

### PR DESCRIPTION
**Pull request by jerodev**
Created a basic php tutorial in dutch

### CODE REVIEW

File: nl-nl/php-nl.html.markdown [hunk 1]
```diff
@@ -0,0 +1,341 @@
+---
+language: PHP
+contributors:
+    - ["Malcolm Fell", "http://emarref.net/"]
+    - ["Trismegiste", "https://github.com/Trismegiste"]
+    - ["Jerodev", "https://github.com/jerodev"]
+filename: learnphp.php
```
Comment (adambard) [inline nl-nl/php-nl.html.markdown]: Needs a `lang` specifier for the human language:

```
lang: nl-nl
```

**Comment by jerodev** [comment]
Added lang specifier

File: nl-nl/yaml-nl.html.markdown [hunk 2]
```diff
@@ -0,0 +1,140 @@
+---
```
Comment (vendethiel) [inline nl-nl/yaml-nl.html.markdown]: can you please remove this file so I can merge your PR?

sorry for the delay

**Comment by vendethiel** [comment]
Going to close this one in the meantime. Ping me if you find some time to fix this (remove the yaml document might be enough)

**Pull request closed without merging**